lib/game.rb

```diff
@@ -0,0 +1,164 @@
+
+class Game
+  
+  attr_accessor :board, :player_1, :player_2
+  
+  WIN_COMBINATIONS = [
+  [0,1,2],
+  [3,4,5],
+  [6,7,8],
+  
+  [0,3,6],
+  [1,4,7],
+  [2,5,8],
+  
+  [0,4,8],
+  [2,4,6]
+  ]
+  
+  
+  def initialize(player_1=Players::Human.new("X"), player_2=Players::Human.new("O"), board=Board.new)
+    @board = board
+    @player_1 = player_1
+    @player_2 = player_2
+  end
+  
+  
+  def board
+    @board
+  end
+  
+  
+  def current_player
+    count = @board.turn_count
+    count.even? ? @player_1 : @player_2
+  end
+    
+    
+  def won?
+    WIN_COMBINATIONS.each do |combo|
+    board = @board.cells
+      wc = board[combo[0]],board[combo[1]],board[combo[2]]
+      
+      if wc.all?{|cell| cell == "X"}
+        puts combo
+        return combo
+      elsif wc.all?{|cell| cell == "O"}
+        puts combo
+        return combo
+      end
+    end
+    false
+  end
+  
+  
+  def draw?
+    if (won? == false && @board.full? == true)
+      puts "Cat\'s Game!"
+      true
+    else
+      false
+    end
+  end
+  
+  
+  def over?
+    won? || draw? ? true : false
+  end
+  
+  
+  def winner
+    
```

Comment:
You could make this method more concise by utilizing the winning combo returned by `won?`:

```ruby
if won = won?
  board.cells[won.first]
end
```